packages/templates/golang.yaml

```diff
@@ -0,0 +1,22 @@
+{{ define "golang_deps" }}
+{{ if .Values.distribution }}
+{{if eq .Values.distribution "opensuse" }}
+- zypper in -y git {{- if .Values.upx -}} upx {{- end }}
+{{else if eq .Values.distribution "fedora" }}
+- dnf install -y git {{- if .Values.upx -}} upx {{- end }}
+{{else if eq .Values.distribution "ubuntu" }}
+- apt-get install -y git {{- if .Values.upx -}} upx {{- end }}
+{{end}}
+{{end}}
+{{end}}
+
+{{ define "golang_download_package" }}
+{{ $version:= get . "version" }}
+{{ $org:= get . "org" }}
+{{ $repo:= get . "repo" }}
+- |
+  PV="{{$version}}" && \
+  PACKAGE_VERSION=${PV%\+*} && \
+  mkdir -p /luetbuild/go/src/github.com/{{$org}} && cd /luetbuild/go/src/github.com/{{$org}} && \
```

Comment:
can we make this more generic? like mayube it doesnt come from github so dest dir shuld not mention github anywhere if possible.